x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/import_rules_route.ts

```diff
@@ -127,8 +134,13 @@ export const importRulesRoute = (
         const [duplicateIdErrors, parsedObjectsWithoutDuplicateErrors] =
           getTupleDuplicateErrorsAndUniqueRules(rules, request.query.overwrite);
 
-        const [nonExistentActionErrors, uniqueParsedObjects] = await getInvalidConnectors(
+        const migratedParsedObjectsWithoutDuplicateErrors = await migrateLegacyActionsIds(
```

Comment:
`Action` here in `migrateLegacyActionIds` refers to connectors right? Wondering if we should have the same language between `migrateLegacyActionIds` and `getInvalidConnectors`. Or is one dealing with the rule action objects and he other dealing with the connectors?

Reply:
This is a similar issue to when rules were known as "alerts" because they were alerting SO's. The security solution calls SO `actions` as **actions** but they are known to kibana alerting as action connectors. My thinking is to follow the same guidance we used between "rules" and "alerts" back when they meant the same thing ('alert' SO types) where we called them rules in the security solution. I believe since the precedent is for connectors to be called 'actions' in the security solution we should continue to use that terminology in the code.